### PR TITLE
Created .travis.yml for package high-scale-lib

### PR DESCRIPTION
**Pull request by rajesh-ibm-power**
Created .travis.yml for package high-scale-lib

### CODE REVIEW

File: travis-ymls/high-scale-lib.travis.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+dist: xenial
+
+arch:
+  - amd64
+  - ppc64le
+
+addons:
+  apt:
+    packages:
+     - maven
```
Comment (Siddhesh-Ghadi) [inline travis-ymls/high-scale-lib.travis.yml:26]: No need to install maven. @rajesh-ibm-power

Comment (rajesh-ibm-power) [reply travis-ymls/high-scale-lib.travis.yml]: @Siddhesh-Ghadi 
Updated with removal of maven addon

**Comment by rajesh-ibm-power** [comment]
@Siddhesh-Ghadi Kindly review and merge

**Comment by Siddhesh-Ghadi** [comment]
Ready for merge @seth-priya

**Pull request merged**